src/ui/SmartSnippet/SmartSnippetSuggestions.ts

```diff
@@ -0,0 +1,88 @@
+import 'styling/_SmartSnippetSuggestions';
+import { Component } from '../Base/Component';
+import { exportGlobally } from '../../GlobalExports';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { QueryEvents, IQuerySuccessEventArgs } from '../../events/QueryEvents';
+import { IPartialQuestionAnswerResponse, IQuestionAnswerResponse } from '../../rest/QuestionAnswerResponse';
+import { $$, Dom } from '../../utils/Dom';
+import { uniqueId, isEqual } from 'underscore';
+import { SmartSnippetCollapsibleSuggestion } from './SmartSnippetCollapsibleSuggestion';
+import { l } from '../../strings/Strings';
+
+const BASE_CLASSNAME = 'coveo-smart-snippet-suggestions';
+const QUESTIONS_LIST_CLASSNAME = `${BASE_CLASSNAME}-questions`;
+const QUESTIONS_LIST_TITLE_CLASSNAME = `${QUESTIONS_LIST_CLASSNAME}-title`;
+
+export interface ISmartSnippetSuggestionsOptions {}
+
+export class SmartSnippetSuggestions extends Component {
+  static ID = 'SmartSnippetSuggestions';
+
+  static options: ISmartSnippetSuggestionsOptions = {};
+
+  static doExport = () => {
+    exportGlobally({
+      SmartSnippetSuggestions
+    });
+  };
+
+  private readonly titleId = uniqueId(QUESTIONS_LIST_TITLE_CLASSNAME);
+  private contentLoaded: Promise<any>;
```

Comment:
Similar to the above comment, should this promise resolve a `boolean`?

Reply:
Same answer as above: the content is always loaded unless a timeout is added.